rules/azure/persistence_azure_automation_webhook_created.toml

```diff
@@ -27,7 +27,6 @@ rule_id = "e9ff9c1c-fe36-4d0d-b3fd-9e0bf4853a62"
 severity = "low"
 tags = ["Elastic", "Cloud", "Azure", "Continuous Monitoring", "SecOps", "Configuration Audit"]
 timestamp_override = "event.ingested"
-to = "now-25m"
```

Comment:
What's the reason for this change? 🤔

Reply:
That appears to be a mistake, it already has `from = "now-25m"` with the default run interval. Accidental addition. 

![image](https://user-images.githubusercontent.com/56412096/114730167-9d5aab00-9d0e-11eb-9213-8ac8bac1df10.png)